relay-server/src/services/processor/profile.rs

```diff
@@ -126,6 +136,25 @@ pub fn expand_profile(
     (profile_id, item_action)
 }
 
+#[cfg(feature = "processing")]
+pub fn expand_profile_chunk(item: &mut Item, config: &Config) -> ItemAction {
+    match relay_profiling::expand_profile_chunk(&item.payload()) {
+        Ok(payload) => {
+            if payload.len() <= config.max_profile_size() {
+                item.set_payload(ContentType::Json, payload);
+                ItemAction::Keep
+            } else {
+                ItemAction::Drop(Outcome::Invalid(DiscardReason::Profiling(
```

Comment:
We should update `Item::quantity()` to avoid creating outcomes that default to `quantity=1`:

https://github.com/getsentry/relay/blob/b6211aaa94887f57f34505d0cd7c0f8c8f6d5f2f/relay-server/src/envelope.rs#L646-L651

Reply:
I don't know if we have a good quantity to set here. Since quantity is now the duration of a profile, it means we need to first parse and process the profile properly, and since Relay won't emit `Accepted` outcomes for profile chunks and only invalid ones, how useful it is to try to process the profile to determine the appropriate duration we failed to ingest?

It will make some charts look a bit weird since it won't be the same unit for valid and invalid profile chunks. I could still try to get a duration but we would still have cases where the profile chunk is malformed or sending bogus data and we can't determine the duration of the profile chunk.

Reply:
After a short discussion with @Dav1dde, I believe the only correct solution is having two different outcomes: one for the number of profile chunks and one for the duration. Not necessarily a blocker for this PR, but I think it's a change we should make.

Reply:
https://github.com/getsentry/relay/pull/3303